localparticipant.go

```diff
@@ -379,3 +372,13 @@ func (p *LocalParticipant) getLocalPublication(sid string) *LocalTrackPublicatio
 	}
 	return nil
 }
+
+func (p *LocalParticipant) onTrackMuted(pub *LocalTrackPublication, muted bool) {
+	if muted {
+		p.Callback.OnTrackMuted(pub, p)
+		p.roomCallback.OnTrackMuted(pub, p)
+	} else {
+		p.Callback.OnTrackUnmuted(pub, p)
+		p.roomCallback.OnTrackUnmuted(pub, p)
+	}
```

Comment:
Is there a chance of this racing and setting wrong callbacks?

Reply:
No, the callback is set in the `CreateRoom` function.